src/inputs/Table/Table.js

```diff
@@ -246,8 +246,11 @@ export const Table = (props) => {
   }, [gridRef, onAddRow]);
 
   const [isRowSelected, setIsRowSelected] = useState(false);
+  if (gridRef?.current?.api?.getSelectedRows().length === 0 && isRowSelected === true) {
```

Comment:
Missing optional chaining operator before `.length` in `gridRef?.current?.api?.getSelectedRows().length` 
_Idem_ in line 253

Reply:
Does _getSelectedRows_ function not return at least a empty array ?

Reply:
The problem is not with `getSelectedRows` but with the chain of _optional chaining_ operators :slightly_smiling_face: 
If `gridRef?.current` is `undefined`, then  `gridRef?.current?.api?.getSelectedRows()` will be `undefined` too, and the code will end up trying to read `undefined.length` and crash.
You can try running the code below to see the error:
```
const obj = {}; // declare an empty object
console.log(obj?.notDefined); // will print "undefined"
console.log(obj?.notDefined.length); // will throw an error because it's trying to read a "length" property from "undefined"
```

Reply:
Nevermind, it seems to work :slightly_smiling_face: